### PR TITLE
Add pvcarrera to contributing maintainers list

### PR DESCRIPTION
**Pull request by pvcarrera**

### CODE REVIEW

File: languages/ruby/maintainers.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+- Pablo Vicente (Slack: `@pvcarrera`, GitHub: `@pvcarrera`)
```
Comment (tehsphinx) [inline languages/ruby/maintainers.md]:
```suggestion
- Pablo Vicente (Slack: `@pvcarrera`, GitHub: `[pvcarrera](https://github.com/pvcarrera)`)
```
Adjusting it to the style in the csharp repo.

Comment (pvcarrera) [reply languages/ruby/maintainers.md]: 👍

Comment (tehsphinx) [reply languages/ruby/maintainers.md]: Can you apply the change?

Comment (pvcarrera) [reply languages/ruby/maintainers.md]: No, I don't have permissions

Comment (tehsphinx) [reply languages/ruby/maintainers.md]: I mean this button above: "Commit suggestion"
image

Comment (pvcarrera) [reply languages/ruby/maintainers.md]: I can add another commit with your suggestion but I can't apply your suggestion on GitHub

**Pull request merged**